charts/lws/README.md

```diff
@@ -0,0 +1,67 @@
+# lws's helm chart
+
+## Table of Contents
+
+<!-- toc -->
+- [Installation](#installation)
+    - [Prerequisites](#prerequisites)
+    - [Installing the chart](#installing-the-chart)
+        - [Install chart using Helm v3.0+](#install-chart-using-helm-v30)
+        - [Verify that controller pods are running properly.](#verify-that-controller-pods-are-running-properly)
+    - [Configuration](#configuration)
+<!-- /toc -->
+
+### Installation
+
+Quick start instructions for the setup and configuration of lws using Helm.
+
+#### Prerequisites
+
+- [Helm](https://helm.sh/docs/intro/quickstart/#install-helm)
+
+#### Installing the chart
+
+##### Install chart using Helm v3.0+
+
+```bash
+$ git clone git@github.com:kubernetes-sigs/kueue.git
+$ cd charts
+$ helm install lws lws --create-namespace --namespace lws-system
+```
+
+##### Verify that controller pods are running properly.
+
+```bash
+$ kubectl get deploy -n kueue-system
```

Comment:
```suggestion
$ kubectl get deploy lws-system
```

Reply:
sorry，I referenced the queue project during development because the helm charts for queue and lws are quite similar.  I've corrected the submission now. The helm chart has been tested and appears to be functioning correctly.